package.json

```diff
@@ -46,8 +52,11 @@
       "mocha": true
     },
     "rules": {
-      "semi": ["error", "always"],
-      "space-before-function-paren": ["error", "never"]
+      "no-param-reassign": "off",
+      "no-underscore-dangle": "off",
+      "no-console": "off",
+      "new-cap": "off",
+      "strict": "off"
```

Comment:
oddly, removing strict caused a bug!?  I read up a bit, but not sure why, since it was there, and I like using on all of my files, I stopped researching and added this.